pkg/ocm/accounts_mgmt.go

```diff
@@ -0,0 +1,84 @@
+package ocm
+
+import (
+	"context"
+
+	"github.com/go-openapi/strfmt"
+	amgmtv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+	"github.com/openshift/assisted-service/pkg/commonutils"
+	"github.com/pkg/errors"
+)
+
+const (
+	ProductCategoryAssistedInstall   = "AssistedInstall"
+	ProductIdOCP                     = "OCP"
+	StatusActive                     = "Active"
+	clusterAuthorizationsPostRequest = "ClusterAuthorizationsPost"
+	subscriptionPatchRequest         = "SubscriptionPatch"
+)
+
+//go:generate mockgen -package ocm -destination mock_accounts_mgmt.go . OCMAccountsMgmt
+type OCMAccountsMgmt interface {
+	CreateSubscription(ctx context.Context, clusterID strfmt.UUID) (*amgmtv1.Subscription, error)
+	UpdateSubscription(ctx context.Context, subscriptionID, openshiftClusterID strfmt.UUID) (*amgmtv1.Subscription, error)
+}
+
+type accountsMgmt struct {
+	client *Client
+}
+
+func (a accountsMgmt) CreateSubscription(ctx context.Context, clusterID strfmt.UUID) (*amgmtv1.Subscription, error) {
+	defer commonutils.MeasureOperation("OCM-CreateClusterAuthorization", a.client.log, a.client.metricsApi)()
+
+	//create the request
+	request, err := amgmtv1.NewClusterAuthorizationRequest().
+		AccountUsername(UserNameFromContext(ctx)).
+		ProductCategory(ProductCategoryAssistedInstall).
+		ProductID(ProductIdOCP).
+		ClusterID(clusterID.String()).
+		Managed(false).
```

Comment:
false is for ocp?

Reply:
False mean that there is no manipulation in AMS.
A manipulation example can be that AMS sees a subscription with no OpenShift cluster ID so it query CS for it.
It our case (assisted-service) we don't use external tools (like have) and everything is done manually - so we don't want/need and manipulation from AMS.

I will guest and say that OCP cluster aren't managed either (in comparison to OSD clusters)